app/views/partials/alerts/list_alarm_callbacks.scala.html

```diff
@@ -27,7 +27,11 @@
 
             <h3>
                 <i class="icon icon-ellipsis-vertical"></i>
-                @availableAlarmCallbacks.get(callback.getType).name
+                @if(availableAlarmCallbacks.get(callback.getType) != null) {
```

Comment:
In this case it's simply a hash, so that's fine. I'm not sure it's possible to store things in variables in scala templates, so this should be ok bug ugly.